### ISSUE TITLE
Error in DayPickerComponent - caused by: not implemented

### ISSUE THREAD

**Issue by artdias90** [valor-software/ngx-bootstrap#2163]
- [x] bug report

`"@angular/common": "2.4.10",`
`"ngx-bootstrap": "1.7.1"`
`"webpack": "2.2.0",`

While trying to add a `datepicker`, I face the following error:
`Error: Uncaught (in promise): Error: Error in ./DayPickerComponent class DayPickerComponent - inline template:9:16 caused by: not implemented`

Calendar Template get's partially loaded, so `imports` seems to be all ok.
Looked everywhere but couldn't find more info.

Other components (using modal and Timepicker) works fine.

note: using webworkers: `"@angular/platform-webworker": "^2.0.0",`

**Comment by bmayen** [comment]
Guessing the problem is in `utils/ng2-bootstrap-config`

```
export function isBs3(): boolean {
  return window.__theme !== 'bs4';
}
```

**Comment by artdias90** [comment]
Actually the problem was it uses `innerHTML`:
```
DayPickerComponent.decorators: [
[innerHTML]=\"CURRENT_THEME_TEMPLATE.ARROW_LEFT\"
...
[innerHTML]=\"CURRENT_THEME_TEMPLATE.ARROW_RIGHT\"
...
]

**Comment by artdias90** [comment]
#2166

**Comment by valorkin** [comment]
fix landed in v1.8
